tools/debugging/stress_test_transfers.py

```diff
@@ -553,6 +561,15 @@ def main() -> None:
                 "--accept-disclaimer",
                 "--log-json",
                 "--disable-debug-logfile",
+                "--flat-fee",
+                "0xf9BA8aDF7F7024D7de8eB37b4c981CFFe3C88Ea7",
+                "0",
+                "--proportional-fee",
+                "0xf9BA8aDF7F7024D7de8eB37b4c981CFFe3C88Ea7",
+                "0",
+                "--proportional-imbalance-fee",
+                "0xf9BA8aDF7F7024D7de8eB37b4c981CFFe3C88Ea7",
```

Comment:
Can we come up with a better way to put the token address in here?

Reply:
yes, but could that be done in a follow up? it just a matter of getting this from the ini file when we need it.

Reply:
Fine for me.